deploy/helm/sumologic/values.yaml

```diff
@@ -316,6 +316,9 @@ fluentd:
     output: false
 
   metadata:
+    ## Option to control capturing of annotations by metadata filter plugin.
+    annotation_match:
+      - "\"sumologic\\.com.*\""
```

Comment:
Hmm do we need to escape `\` ?

Reply:
`.` must be escaped in regular expression, `\` must be escaped in yaml

e.g. when I changed expression to this form ` "\".*sumologic\.com.*\""` there is this `error: error converting YAML to JSON: yaml: line 17: found unknown escape character`

Reply:
could we use `'"sumologic.com*"'`?

Reply:
`'"sumologic.com\\.*"'` -> wrong dot is escaped, I can test with this form `'"sumologic\\.com.*"'`

Reply:
@sumo-drosiek I've changed to this form: `'"sumologic\\.com.*"'`